style.css

```diff
@@ -0,0 +1,122 @@
+html {
+  margin: 0;
+}
+
+body {
+  width: 100vw;
+  margin-top: 0;
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+  height: 5410px;
```

Comment:
This is huuuuugggee 😮 I don't think we need all of that, how about we tell the browser to just take `100%` ? It would be then:
```
height: 100%;
```